src/app/app.module.ts

```diff
@@ -1,18 +1,41 @@
 import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule }   from '@angular/forms';
 import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
 
 import { AppComponent } from './app.component';
-import { CoreComponent } from './core/core.component';
+import { VehiclesManagementComponent } from './feature/vehicles-management/vehicles-management.component';
+import { VehiclesService } from './core/vehicles/vehicles.service';
+import { VehiclesRestService } from './core/vehicles/vehicles-rest.service';
```

Comment:
Skorzystaj z aliasowania. #2 -> https://medium.com/@tomastrajan/6-best-practices-pro-tips-for-angular-cli-better-developer-experience-7b328bc9db81